mathesar/tests/integration/test_schemas.py

```diff
@@ -0,0 +1,18 @@
+from playwright.sync_api import Page, expect
+
+
+def test_create_and_delete_schema(page: Page, schemas_page_url):
+    page.goto(schemas_page_url)
+    schema_name = "foo"
+    schema_entry = page.locator(f".schema-list .schema-row:has-text('{schema_name}')")
+    expect(schema_entry).not_to_be_visible()
+    page.click("text=New Schema")
+    page.fill("[aria-label='name']", schema_name)
+    page.click("button:has-text('Save')")
+    expect(schema_entry).to_be_visible()
+    # We're also deleting the schema in the same test as a way of cleaning up
+    # the state created in this test so as not to interfer with other tests.
+    # This is a hack for now.
+    schema_entry.locator("button[aria-label='Delete Schema']").click()
```

Comment:
I agree that this is hacky. Can we clean it up in a subsequent PR? Being that I've already [spent some time](https://github.com/centerofci/mathesar/pull/1062#issuecomment-1043357864) trying to find a better solution to this hack, how would you feel about writing the fixture for it yourself since I think you'd be able to get it working quicker than I could?